src/GovUk.Education.ExploreEducationStatistics.Public.Data.Api/Controllers/DataSetsController.cs

```diff
@@ -38,6 +38,25 @@ public DataSetsController(IDataSetService dataSetService)
             .HandleFailuresOrOk();
     }
 
+    /// <summary>
+    /// Get a data set version
+    /// </summary>
+    /// <remarks>
+    /// Get a data set version, including a full list of its changes.
+    /// </remarks>
+    [HttpGet("{dataSetId:guid}/versions/{dataSetVersion}")]
+    [Produces("application/json")]
+    [SwaggerResponse(200, "The requested data set version", type: typeof(DataSetVersionViewModel))]
+    [SwaggerResponse(404)]
+    public async Task<ActionResult<DataSetVersionViewModel>> GetVersion(
+        [SwaggerParameter("The ID of the data set.")] Guid dataSetId,
+        [SwaggerParameter("The data set version.")] string dataSetVersion)
```

Comment:
Might be worth throwing some little examples into the description e.g. `The data set version e.g. 1.0, 1.1, 2.0, etc`